src/akamai/lib/AkamaiHDPlayer.as

```diff
@@ -297,6 +323,21 @@ package
      */
     public function onChangeVolume(event:DynamicEvent):void
     {
+      var volume:Number = (Number)(event.args);
+      _streamController.volume = volume;
+      //Dispatches the VOLUME_CHANGED event only when the change occures properly.
+      if (_streamController.volume == volume)
```

Comment:
small question - are we okay that it will raise the event if it was already that volume?  I think it's okay but just wondering

@chibzu